src/main/java/ch/njol/skript/effects/EffPlaySound.java

```diff
@@ -132,16 +135,25 @@ public String toString(@Nullable Event event, boolean debug) {
 			.append("play sound ")
 			.append(sounds.toString(event, debug));
 		
-		if (category != null) builder.append(" in ")
-			.append(category.toString(event, debug));
-		if (volume != null) builder.append(" with volume ")
-			.append(volume.toString(event, debug));
-		if (pitch != null) builder.append(" with pitch ")
-			.append(pitch.toString(event, debug));
-		if (locations != null) builder.append(" at ")
-			.append(locations.toString(event, debug));
-		if (players != null) builder.append(" to ")
-			.append(players.toString(event, debug));
+		if (category != null) {
+			builder.append(" in ").append(category.toString(event, debug));
+		}
+		
+		if (volume != null) {
+			builder.append(" with volume ").append(volume.toString(event, debug));
+		}
+		
+		if (pitch != null) {
+			builder.append(" with pitch ").append(pitch.toString(event, debug));
+		}
+		
+		if (locations != null) {
+			builder.append(" at ").append(locations.toString(event, debug));
+		}
+		
+		if (players != null) {
+			builder.append(" to ").append(players.toString(event, debug));
+		}
```

Comment:
Singular statement conditions can have their braces omitted
```suggestion
		if (category != null)
			builder.append(" in ").append(category.toString(event, debug));
		
		if (volume != null)
			builder.append(" with volume ").append(volume.toString(event, debug));
		
		if (pitch != null)
			builder.append(" with pitch ").append(pitch.toString(event, debug));
		
		if (locations != null)
			builder.append(" at ").append(locations.toString(event, debug));
		
		if (players != null)
			builder.append(" to ").append(players.toString(event, debug));
```